docs/examples/promise/07_transfer_dots/index.js

```diff
@@ -1,7 +1,7 @@
 // Import the API, Keyring and some utility functions
 const { ApiPromise } = require('@polkadot/api');
 const { Keyring } = require('@polkadot/keyring');
-const { stringToU8a } = require('@polkadot/util');
+const { stringShorten, stringToU8a } = require('@polkadot/util');
```

Comment:
Not sure why we are doing this. The text changes helps, the extra "show off what utils we have" doesn't really. So keep to what it was.